cypress/e2e/tests/pages/manager/cluster-manager.spec.ts

```diff
@@ -156,6 +159,10 @@ describe('Cluster Manager', { testIsolation: 'off', tags: ['@manager', '@adminUs
         cy.get('[data-testid="clusterBasics__noneOptionSelectedForCni"]').should('exist');
         // EO test for https://github.com/rancher/dashboard/issues/10338 (added option 'none' for CNI)
 
+        labeledSelectPo.toggle();
+        labeledSelectPo.clickLabel('calico');
+        labeledSelectPo.checkOptionSelected('calico');
```

Comment:
this should be a po in `BasicsRke2` 

```
networks(): LabeledSelectPo {
    return new LabeledSelectPo('[data-testid="cluster-rke2-cni-select"]');
  }
```

then import into `ClusterManagerEditRke2CustomPagePo`

```
  basicsTab(): BasicsRke2 {
    return new BasicsRke2();
  }
```

and then use it in your test
  ```
  editCreatedClusterPage.basicsTab().networks().toggle();
  editCreatedClusterPage.basicsTab().networks().clickOptionWithLabel('calico');
  editCreatedClusterPage.basicsTab().networks().checkOptionSelected('calico');
```